.github/workflows/e2e-junit.yml

```
@@ -0,0 +1,26 @@
+name: E2E Tests with JUnit
+
+on: [push]
```

This looks to be running the E2E tests twice. Once for push and one for pull request. Should adjust it to be like the others.

```suggestion
on:
  pull_request:
    branches: [ master, release/** ]
  push:
    branches: [ master, release/** ]
    tags: [ v* ]
```